src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs

```diff
@@ -20,9 +43,158 @@ public class MauiMediaElement : Grid, IDisposable
 	public MauiMediaElement(MediaPlayerElement mediaPlayerElement)
 	{
 		this.mediaPlayerElement = mediaPlayerElement;
+
+		btn = new Button
+		{
+			Background = new Microsoft.UI.Xaml.Media.SolidColorBrush(Microsoft.UI.Colors.Transparent),
+			Foreground = new Microsoft.UI.Xaml.Media.SolidColorBrush(Microsoft.UI.Colors.Transparent),
+			Width = 40,
+			Height = 40
+		};
+		btn.Click += ButtonSetFullScreenStatus;
+
+		Image image = new()
+		{
+			Width = 40,
+			Height = 40,
+			HorizontalAlignment = Microsoft.UI.Xaml.HorizontalAlignment.Right,
+			VerticalAlignment = Microsoft.UI.Xaml.VerticalAlignment.Top,
+			Source = new BitmapImage(new Uri("ms-appx:///whitefs.png")),
+		};
+
+		ButtonContainer = new()
+		{
+			HorizontalAlignment = Microsoft.UI.Xaml.HorizontalAlignment.Right,
+			VerticalAlignment = Microsoft.UI.Xaml.VerticalAlignment.Top,
+			Width = 40,
+			Height = 40
+		};
+		ButtonContainer.Children.Add(image);
+		ButtonContainer.Children.Add(btn);
+
 		Children.Add(this.mediaPlayerElement);
+		Children.Add(ButtonContainer);
+
+		mediaPlayerElement.PointerMoved += MediaPlayerElement_PointerMoved;
```

Comment:
rename to align on C# parttern.
```suggestion
		mediaPlayerElement.PointerMoved += OnMediaPlayerElementPointerMoved;
```